libs/expo/betterangels/src/lib/apollo/graphql/queries.ts

```diff
@@ -101,6 +101,9 @@ export const GET_NOTE = gql`
         email
         firstName
         lastName
+        clientProfile {
+          pk
```

Comment:
[not blocking] just a question (for another day maybe), but should we rename  pk to id?

Reply:
aliased `pk` to `id` for consistency